src/endpoints/account-authentication-settings.js

```diff
@@ -0,0 +1,14 @@
+import CRUDExtend from '../extends/crud'
+
+class AccountAuthenticationSettingsEndpoint extends CRUDExtend {
```

Comment:
If this class extends from CRUDExtend then it will have create/delete/update methods. If that was intentional and backend support those methods then you need to add them in `.d.ts` file as well as in tests. you can expose it manually in `.d.ts` file (like in case of `SettingsEndpoint`) or you can extend `CrudQueryableResource`. Take a look at other `.d.ts` files for examples.